packages/medulas-react-components/src/components/forms/TextFieldForm/index.tsx

```diff
@@ -22,6 +22,7 @@ const TextFieldForm = ({ name, form, validate, ...restProps }: Props): JSX.Eleme
     <MuiTextField
       error={error}
       name={input.name}
+      value={input.value}
```

Comment:
If you want to set initial value for the form you can use `initalValues` as well during form initialization. https://github.com/final-form/react-final-form-hooks/issues/2#issuecomment-438323339

Reply:
I tried this but it was not working in our setup. I don't understand why. However, an initial values is not needed at the moment as long as the user input is preserved.